packages/engine/src/mocap/MotionCaptureSystem.ts

```diff
@@ -138,16 +138,24 @@ const execute = () => {
     for (const boneName of VRMHumanBoneList) {
       const localbone = rigComponent.localRig[boneName]?.node
       if (!localbone) continue
-      if (!MotionCaptureRigComponent.lowerBodySolveFactor[entity]) {
-        if (
-          boneName == VRMHumanBoneName.LeftUpperLeg ||
-          boneName == VRMHumanBoneName.RightUpperLeg ||
-          boneName == VRMHumanBoneName.LeftLowerLeg ||
-          boneName == VRMHumanBoneName.RightLowerLeg ||
-          boneName == VRMHumanBoneName.LeftFoot ||
-          boneName == VRMHumanBoneName.RightFoot
+      if (
+        boneName == VRMHumanBoneName.LeftUpperLeg ||
+        boneName == VRMHumanBoneName.RightUpperLeg ||
+        boneName == VRMHumanBoneName.LeftLowerLeg ||
+        boneName == VRMHumanBoneName.RightLowerLeg ||
+        boneName == VRMHumanBoneName.LeftFoot ||
+        boneName == VRMHumanBoneName.RightFoot
+      ) {
+        localbone.quaternion.slerp(
```

Comment:
we have fastSlerp injected onto the quaternion class which is preferable to slerp